jekyll/_cci2/env-vars.md

```diff
@@ -493,16 +493,13 @@ For more details, see [Setting an Environment Variable in a Shell Command](#sett
 Variable                    | Type    | Value
 ----------------------------|---------|-----------------------------------------------
 `CI`                        | Boolean | `true` (represents whether the current environment is a CI environment)
```

Comment:
Is this deprecated?

Reply:
`CI` I mean

Reply:
no, this is fairly common across multiple providers - that's why we have `CI` and `CIRCLECI`